docs/package/gluon-ssid-changer.md

```diff
@@ -0,0 +1,70 @@
+gluon-ssid-changer
+==================
+
+This package adds a script to change the SSID to an Offline-SSID when there is
+no connection to any gateway.
+This SSID can be generated from the nodes hostname with the first
+and last part of the nodename or the mac address, to allow observers to 
+recognise which node is down. The script is called once a minute by micron.d
+and it will change from online to offline-SSID maximum once every (definable)
+timeframe.
+
+You can enable/disable it in the config mode.
+
+Once every timeframe it checks if there's still a gateway reachable. There can be
+selected two algorithms to analyze if a gateway is reacheable:
+
+- `tq_limit_enabled=true`: (not working with BATMAN_V) define an upper and lower
+  limit to turn the offline_ssid on and off in-between these two values the SSID
+  will never be changed to prevent it from toggeling every minute.
+- `tq_limit_enabled=false`: there will be only checked, if the gateway is
+  reachable with:
+
+        batctl gwl -H
+
+Depending on the connectivity, it will be decided if a change of the SSID is 
+necessary: There is a variable `switch_timeframe` (for ex.  1440 = 24h) that 
+defines a time interval after which a successful check that detects an offline
+state will result in a single change of the SSID to "FF_Offline_$node_hostname".
```

Comment:
The parameter `switch_timeframe` defines the period of time after which gateways are considered unreachable resulting in the change of the SSID to "FF_Offline_$node_hostname" (in minutes).

Reply:
This is not how it works.

the timeframe is just for measurement. during that timeframe, every minute it is recorded, if the gateway is reacheable or not. If half of the checks result in not-reacheable, the SSID will be changed to the Offline SSID

Reply:
I should have looked at the code first. Maybe I still misunderstand how it works. If you have a 24 hour switch_timeframe and the first 12 hours every check is successful, doesn't it take another 12 hours of failure until the failure state is recognized and vice versa?

Reply:
yes, as you wish. we recomment a 60 minutes timeframe, so nodes don't go into offline mode too lightly

Reply:
I renamed to rest, and rephrazed the whole block.